src/poll.ts

```diff
@@ -236,17 +250,36 @@ namespace Poll
 
 			return null;
 		}
+		
+		let userCount: number = 0;
+		for (let member of server.members.array())
```

Comment:
We probably shouldn't count bots in this. The boolean `User.bot` is true if it is a bot (duh). Make sure that you get the `User`, not the `GuildMember`. If you have the `GuildMember` and want the `User`, you can use `GuildMember.user`.

Reply:
why use the user for counting instead of guildmember, how is this better?

Reply:
It's not that you're using the `User` for counting, it's just that the `bot` boolean is a part of `User`, not `GuildMember`. So in order to make sure the user is not a bot, you have to get the `User`.